todo_app/templates/index.html

```diff
@@ -9,10 +9,21 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2>Task Master</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in get_items() %}
+        <li>{{item.title}}</li>
+         {% endfor %}
+          </div>
+        
       </ul>
+      <!-- setting the URL path for where the form is to be 'posted' to -->
+      <form action= "/newitem" method= "post">
```

Comment:
not even a minor point, just sharing something cool, with flask you can tell it to use specific methods, so instead of `action="/newitem"` you can have `action={{ url_for('new_item') }}"`
It's not obvious which is nicer but it's good to know :)